frontend/lib/strapi-sdk/operations/getProductList.graphql

```diff
@@ -15,6 +15,7 @@ query getProductList($filters: ProductListFiltersInput) {
             description
             metaDescription
             metaTitle
+            defaultShowAllChildrenOnLgSizes
```

Comment:
In the past, we've had to deploy Strapi API changes before deploying frontend changes to use that API. Otherwise, in the gap between when Vercel redeploys, and when a dev manually redeploys Strapi, this query will fail because the field doesn't exist and the page will 500 (you'll notice that the [prod deployment](https://react-commerce-prod-git-display-4-rows-devices-on-684edd-ifixit.vercel.app/Tools) for this branch is 500ing right now). This is unfortunate, and we'd like to solve this in the future by automatically redeploying Strapi ([ref](https://github.com/iFixit/ifixit/issues/42822)).

I would suggest making all the Strapi changes in one pull, merging and deploying that pull, then making the frontend changes.

Reply:
You are right, I'll split this PR in two